src/app/static/src/app/components/plots/choropleth/Choropleth.vue

```diff
@@ -417,22 +419,15 @@
                 return {id: node.indicator, label: node.name};
             }
         },
-        watch:
-            {
-                initialised: function (newVal: boolean) {
-                    this.updateBounds();
-                },
-                selectedAreaFeatures: function (newVal) {
-                    this.updateBounds();
-                },
+        watch: {
                 filters: function () {
                     this.initialise();
                 },
                 indicators: function () {
                     this.initialise();
                 },
             },
```

Comment:
Do you not need to watch if the value has changed before calling these functions ?

Reply:
Do you mean the `initialised` and the `selectedAreaFeatures` watchers I have gotten rid of? These I have gotten rid of because the `updateBounds` function is called when the map fires `@ready` and `@vnode-updated` and with those events these watchers become redundant.

Reply:
I guess I got confused, the previous implementation does have some like this `filters: function (newVal: boolean) {this.initialise(); }`

So you basically removed the `newVal` because it was not used, right ?